components/ledger/pkg/api/apierrors/errors.go

```diff
@@ -51,8 +52,6 @@ func coreErrorToErrorCode(err error) (int, string, string) {
 	switch {
 	case runner.IsConflictError(err):
 		return http.StatusConflict, ErrConflict, ""
-	case runner.IsInsufficientFundError(err):
```

Comment:
A reason to remove this error?

Reply:
It's not used anymore

Reply:
Now the only insufficient funds error comes from machine

Reply:
Ok